### ISSUE TITLE
Recommended Citation for HydroShare Jupyter-Hub

### ISSUE THREAD

**Issue by ChristinaB** [hydroshare/hydroshare-jupyterhub#49]
@Castronova and @ddcamiu 
We are writing up a paper where we have a reproducibility section and are describing how we are using HydroShare for a landslide model. Can you let me know how we should be citing you?  What I could find online is below: 

... using Jupyter Notebooks that access Landlab installed on HydroShare JupyterHub servers at the National Center for Supercomputing Applications (Yin, 2016; Castranova, 2017),...  

Yin, D. (2017). JupyterHub-NCSA, HydroShare, http://www.hydroshare.org/resource/80d9f3b4bc914628a2d1df4ebebcc3fd

Castranova, T., 2017, Dockerfile for the HydroShare-JupyterHub base image. https://hub.docker.com/r/castrona/hydroshare-jupyterhub/ Accessed April 4, 2017

I am still planning to write the Landlab-CyberGIS/NCSA-HydroShare paper  on this and will keep you posted.  But for now, I just need whatever citation we have. 
C

**Comment by Castronova** [comment]
I think it is acceptable to cite the object that is stored inside hydroshare since this is where people will go to access the JupyterHub system.  I've been writing a paper about the overall HydroShare JupyterHub software stack which be a good reference for future work. 

It would be great if you could cite this repository too.  Maybe we should formally publish version 1.1 of the JupyterHub code using HydroShare so that we can get a DOI.  You could also follow a this pattern (https://academia.stackexchange.com/questions/14010/how-do-you-cite-a-github-repository): 

```
@misc{Charles2013,
  author = {Charles, P.W.D.},
  title = {Project Title},
  year = {2013},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/charlespwd/project-title}},
  commit = {4f57d6a0e4c030202a07a60bc1bb1ed1544bf679}
}
```

**Comment by Castronova** [comment]
If you decide to cite the Dockerfile...some modifications:

Castronova, Anthony M., 2017, Dockerfile for the HydroShare-JupyterHub base image. https://hub.docker.com/r/castrona/hydroshare-jupyterhub/ Accessed April 4, 2017

**Comment by ChristinaB** [comment]
I think publishing the code to get a DOI is probably the best long term solution to how to cite the HydroShare piece of  JupyterHub, we can post something on a help page on how to cite the use of the server that includes Dandong CyberGIS and also one that is specific to JupyterHub on HydroShare. That would be better than the Dockerfile, right? or both?

**Comment by ddcamiu** [comment]
I submitted another piece of Jupyter work to a conference, which my advisor suggests to cite as NCSA Jupyter effort:

Yin, D., Liu, Y. Y., Padmanabhan, A., Terstriep, J., Rush, J., Wang, S. (2017) A CyberGIS-Jupyter Framework for Geospatial Analytics at Scale. The First Practice & Experience in Advanced Research Computing (PEARC) Conference, Under review

**Comment by ChristinaB** [comment]
@Castronova and @ddcamiu   are there other citations you recommend we include in the paper I am working on ?

**Comment by ChristinaB** [comment]
@ddcamiu - Did the Paper above get submitted?  I am working on writing up my paper with Landlab.

**Comment by ddcamiu** [comment]
Yes. It was published this June. https://dl.acm.org/citation.cfm?id=3093378

**Comment by ChristinaB** [comment]
@Castronova  Is this the current citation  ? https://hub.docker.com/r/castrona/hydroshare-jupyterhub/  It says it hasn't been updated in a year and I'm wondering if we can get a DOI of the image for the KI paper? 
Here's how I have it in the references until I hear otherwise: 
Castronova, Anthony M. 2018. Dockerfile for the HydroShare-JupyterHub base image.  Retrieved October 29, 2018, from https://hub.docker.com/r/castrona/hydroshare-jupyterhub/.
